care.js

```diff
@@ -116,21 +117,54 @@ function doTheCodes() {
   var todayCommits = 0;
   var weekCommits = 0;
 
-  var today = spawn('sh ' + __dirname + '/standup-helper.sh', ['-m ' + config.depth, config.repos], {shell:true});
-  todayBox.content = '';
-  today.stdout.on('data', data => {
-    todayCommits = getCommits(`${data}`, todayBox);
-    updateCommitsGraph(todayCommits, weekCommits);
-    screen.render();
-  });
+  function getCommits(box, data) {
+    var content = colorizeLog(data || '');
+    box.content += content;
+    var commitRegex = /(.......) (- .*)/g;
+    return (box && box.content) ? (box.content.match(commitRegex) || []).length : '0';
+  }
 
-  var week = spawn('sh ' + __dirname + '/standup-helper.sh', ['-m ' + config.depth + ' -d 7', config.repos], {shell:true});
-  weekBox.content = '';
-  week.stdout.on('data', data => {
-    weekCommits = getCommits(`${data}`, weekBox);
-    updateCommitsGraph(todayCommits, weekCommits);
-    screen.render();
-  });
+  if (config.gitbot.toLowerCase() === 'gitstandup') {
+    todayBox.content = '';
+    config.repos.forEach(repo => {
+      var today = spawn(`cd ${repo} && git-standup`, [`-m ${config.depth}`, '-d 1'], {shell:true});
```

Comment:
uhhhhhh so I'm not 100% sure this works, because git-standup is on the wrong path (and works locally, but doesn't work when installed as a package). Can you just revert this, and leave the old git-standup behaviour, so that this PR _just_ adds gitlog, and not other things? That way it's easier if something goes wrong after this PR :)